internal/armador/create.go

```diff
@@ -78,11 +77,11 @@ func Create(cmd commands.Command, projectDir, namespace string, dirs commands.Di
 
 	if len(armadorFiles) < 1 {
 		logger.Warnf(`There's no armador file available at: %s
-    Without an Armador file, it's unclear what values/dependencies to use.
-    Under most cases, if there is only one chart to install,
-    you should just execute 'helm install'.
-
-    If this is not the case than it's possibe the --projectDir (%s) may be wrong.`, projectDir, projectDir)
+		Without an Armador file, it's unclear what values/dependencies to use.
+		Under most cases, if there is only one chart to install,
```

Comment:
this indentation was correct for what is displayed in the terminal - please revert